quickwit-cluster/src/cluster.rs

```diff
@@ -193,10 +207,13 @@ impl Cluster {
         self.members.borrow().clone()
     }
 
-    /// Return the grpc addresses corresponding to the list of members.
-    pub async fn members_grpc_addresses(
+    /// Return the grpc addresses corresponding to the list of members
```

Comment:
Returns the gRPC addresses of the members providing the specified service. Returns the addresses of all the members if no service is specified.